doc/source/lilac/obtaining-building-and-running/obtaining-and-building-ctsm.rst

```diff
@@ -26,7 +26,7 @@ Obtain CTSM by running::
 
   git clone https://github.com/ESCOMP/CTSM.git
   cd CTSM
-  ./manage_externals/checkout_externals
+  ./bin/git-fleximod update
 
 Then build CTSM and its dependencies. On a machine that has been ported to CIME, the
 command will look like this (example given for NCAR's ``cheyenne`` machine)::
```

Comment:
shouldn't we remove cheyenne here?

Reply:
@ekluzek should we expand the scope of this documentation update to replacing references to cheyenne to derecho?

Reply:
@slevis-lmwg you could search for and remove instances of cheyenne (some you'd replace with Derecho). That's not required, but wouldn't be bad to do. It is something we should do at some point.

Issues referencing that are:

#2467
#2223

Reply:
To further clarify:
- I interpreted the above to mean that I should replace cheyenne with derecho in this .rst file for this PR.
- Should I interpret your comment @ekluzek to mean that I should resolve #2467 and #2223 as part of this PR?

Reply:
@slevis-lmwg it's something you *could* do, but aren't required to. It's not something to hold the tag, up for, but if the amount of work looks reasonable and you are up for it, it would be *good* to do. And it is something that we need to make sure we do at some point. You could just replace the one instance. I'm just pointing out that there are a lot of instances to Cheyenne that we need to deal with eventually.

Reply:
To keep things moving, I will not proceed with #2223 
Sorry, Jim, it'll happen later.